tests/test_perseptual.py

```diff
@@ -128,10 +129,69 @@ def test_lpips_loss_forward(input_tensors: Tuple[torch.Tensor, torch.Tensor], de
     loss(prediction.to(device), target.to(device))
 
 
+def test_lpips_loss_backward(prediction: torch.Tensor, target: torch.Tensor, device: str) -> None:
+    loss = LPIPS()
+    loss(prediction.to(device), target.to(device))
+
+
 def test_lpips_loss_raises_if_wrong_reduction(prediction: torch.Tensor, target: torch.Tensor) -> None:
     for mode in ['mean', 'sum', 'none']:
         LPIPS(reduction=mode)(prediction, target)
 
     for mode in [None, 'n', 2]:
         with pytest.raises(KeyError):
             LPIPS(reduction=mode)(prediction, target)
+
+
+# ================== Test class: `DISTS` ==================
+def test_dists_loss_forward(prediction: torch.Tensor, target: torch.Tensor, device: str) -> None:
+    loss = DISTS()
+    loss(prediction.to(device), target.to(device))
+
+
+def test_dists_loss_backward(prediction: torch.Tensor, target: torch.Tensor, device: str) -> None:
+    loss = LPIPS()
+    loss(prediction.to(device), target.to(device))
+
+
+@pytest.mark.parametrize(
+    "prediction,target,expectation,value",
+    [
+        (torch.zeros(4, 3, 128, 128), torch.zeros(4, 3, 128, 128), raise_nothing(), 0.0),
+        (torch.ones(4, 3, 128, 128), torch.ones(4, 3, 128, 128), raise_nothing(), 0.0),
```

Comment:
I assume you wanted to reduce resolution here to 96 as well

Reply:
Yes, missed this lines